app/views/buildings/show.html.erb

```diff
@@ -1,10 +1,47 @@
 <h1 class="building-name"><%= @building.name %></h1>
 <h2>Suites</h2>
-<ul>
-  <% @suites.each do |suite| %>
-    <li><%= link_to suite.number, suite_path(suite) %> - <%= pluralize(suite.size,'bed') %></li>
-  <% end %>
-</ul>
+<% @suites.keys.each do |size| %>
+  <div class="suites-list suites-<%= size_str(size).pluralize %>">
+    <h3><%= size_str(size).pluralize.titleize %></h3>
+    <table>
```

Comment:
The column widths here are all over the place since it adjusts on a table by table basis. I'll add some styling.